src/components/backend-ai-webui-styles.ts

```diff
@@ -334,10 +334,16 @@ export const BackendAIWebUIStyles = [
       --mdc-select-dropdown-icon-color: #747474;
       --mdc-select-focused-dropdown-icon-color: rgba(255, 255, 255, 0.42);
       --mdc-select-disabled-dropdown-icon-color: #747474;
+      /* Need to be set when fixedMenuPosition attribute is enabled */
+      --mdc-menu-max-width: 300px;
+      --mdc-menu-min-width: 300px;
```

Comment:
Same here.
<img width="418" alt="image" src="https://user-images.githubusercontent.com/28584164/178414040-a7b8b5bf-d959-47dd-a691-d3707acf0d5c.png">

Reply:
hmm... holding...

Reply:
fixed it